docs/src/openapi-v1.yaml

```
@@ -747,6 +748,187 @@ paths:
+        Returns an overview of the current of the current auction. There is only
+        one auction at a time. If there is no auction most fields will be `null`.
+      parameters:
+      - name: at
+        in: query
+        description: Block at which to retrieve info at.
+        required: false
+        schema:
+          type: string
+          description: Block identifier, as the block height or block hash.
+          format: unsignedInteger or $hex
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ParasAuctionsCurrent'
+  /experimental/paras/crowdloans:
+    get:
+      tags:
+      - paras
+      summary: |
+        [Experimental subject to breaking change] List all available crowdloans.
+      description: |
+        Returns a list of all the `paraId`s that have a crowdloan. Can optionally
+        specify to include the `fundInfo`.
+      parameters:
+      - name: at
+        in: query
+        description: Block at which to retrieve info at.
+        required: false
+        schema:
+          type: string
+          description: Block identifier, as the block height or block hash.
+          format: unsignedInteger or $hex
+      - name: fundInfo
+        in: query
+        description: Wehther or not to include the `fundInfo` of each crowdlaon
+        required: false
+        schema:
+          type: boolean
+          default: false
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ParasCrowdloans'
+                
+  /experimental/paras/{paraId}/crowdloan-info:
+    get:
+      tags:
+      - paras
+      summary: |
+        [Experimental subject to breaking change] Get crowdloan information for a
+        `paraId`
+      description: |
+        Returns `fundInfo` and the set of `leasePeriods` a crowdloan covers.
+      parameters:
+      - name: paraId
+        in: path
+        description: paraId to query crowdloan information of.
+        required: true
+        schema:
+          type: number
+      - name: at
+        in: query
+        description: Block at which to retrieve info at.
+        required: false
+        schema:
+          type: string
+          description: Block identifier, as the block height or block hash.
+          format: unsignedInteger or $hex
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ParasCrowdloanInfo'
+  /experimental/paras/{paraId}/lease-info:
+    get:
+      tags:
+      - paras
+      summary: |
+        [Experimental subject to breaking change] Get current and future lease
+        info + lifecycle stage for a given `paraId.
```

```suggestion
        info as well as the lifecycle stage for a given `paraId.
```

Optional suggestion. Not sure if having symbols is conventional for docs.